server/src/option.rs

```diff
@@ -72,6 +73,22 @@ impl Config {
         }
     }
 
+    pub async fn validate_storage(&self, storage: &impl ObjectStorage) {
+        match storage.check().await {
+            Ok(_) => (),
+            Err(ObjectStorageError::NoSuchBucket(name)) => panic!(
+                "Could not start because the bucket named {bucket} doesn't exist. Please make sure bucket with the name {bucket} exists on {url} and then start parseable again",
+                bucket = name,
```

Comment:
{bucket} is repeated. Make it like

```
"Could not start because the bucket doesn't exist. Please ensure bucket {bucket} exists on {url}. Then start Parseable."
```